index.js

```diff
@@ -1,24 +1,22 @@
-var os = require('os')
-var path = require('path')
+var os = require("os");
```

Comment:
Please revert these changes. They make it really hard to find out later when & how something was changed.